api/v1alpha3/zz_generated.conversion.go

```diff
@@ -164,11 +164,6 @@ func RegisterConversions(s *runtime.Scheme) error {
 	}); err != nil {
 		return err
 	}
-	if err := s.AddGeneratedConversionFunc((*OpenStackMachineSpec)(nil), (*v1alpha4.OpenStackMachineSpec)(nil), func(a, b interface{}, scope conversion.Scope) error {
```

Comment:
This is a pretty huge diff.. Shouldn't include so many changes. Can you try to find out why that is?